components/location/popup.jsx

```diff
@@ -0,0 +1,319 @@
+/* eslint-disable react/prop-types */
+import React, { useContext } from "react";
+
+import { LanguageContext } from "@contexts/translator";
+
+///////////
+// returns a an array of urls as strings if there is a match otherwise null
+// e.g. ['https://google.com']
+///////////
+function extractUrls(item) {
```

Comment:
Definitely open to alternatives here if you said you had some ideas!

Reply:
Yeah this is... a lot haha but let's revisit this in a different PR! Depending on how the Airtable data looks, I like the idea of using a markdown parsing library here instead.